modules/cloud-guides/pages/integrate-teradata-jupyter-extensions-with-google-vertex-ai.adoc

```diff
@@ -0,0 +1,162 @@
+jupyter labextension install --no-build teradata_preferences*
+ 
+ 
+# PIP install the Teradata Python & R libraries
+pip install teradataml
+Rscript -e "install.packages('tdplyr',repos=c('https://teradata-download.s3.amazonaws.com','https://cloud.r-project.org'))"
+
+# rebuild jupyter lab 
+jupyter lab build --dev-build=False --minimize=False
+----
+
+=== 4. Create a new notebook and add the startup script from cloud storage bucket 
+
+image::vertex.create.notebook.startupscript.png[create a new notebook with startup script, width=100%]
+
+It may take a few minutes for the notebook creation process to complete. When it is done, click on 'Open notebook'.
+
+image::vertex.open.notebook.png[Open notebook, width=100%]
+
+== Use custom container
+
+Another option is to provide a custom container when creating a notebook.
+
+=== 1. Download Teradata Jupyter extensions package
+Go to link:https://downloads.teradata.com/download/tools/vantage-modules-for-jupyter[Vantage Modules for Jupyter] page to download the Teradata Jupyter extensions package bundle Linux version.
+
+Copy this package to your work directory and unzip it.
+
+=== 2. Build custom Docker image
+The custom container must expose a service on port 8080. It is recommended to create a container derived from a Google Deep Learning Containers image, because those images are already configured to be compatible with user-managed notebooks.
+
+Below is a sample Dockerfile you can use to build a Docker image with Teradata SQL kernel and extensions installed:
+
+[source, Dockerfile, role="content-editable"]
+----
+# Use one of the deep learning images as base image
+# if you need both Python and R, use one of the R images
+FROM gcr.io/deeplearning-platform-release/r-cpu:latest
+ 
+USER root
+ 
+##############################################################
+# Install kernel and copy supporting files
+##############################################################
+ 
+# Copy the kernel
+COPY ./teradatakernel /usr/local/bin
+ 
+RUN chmod 755 /usr/local/bin/teradatakernel
+ 
+# Copy directory with kernel.json file into image
+COPY ./teradatasql teradatasql/
+ 
+# Copy notebooks
+COPY ./ShippedNotebooks/ /home/jupyter
+COPY ./StartHere.ipynb ./license.txt /home/jupyter
+ 
+# Install the kernel file to /opt/conda jupyter lab instance
+RUN jupyter kernelspec install ./teradatasql --prefix=/opt/conda 
+ 
+ 
+##############################################################
+# Install Teradata extensions
+##############################################################
+ 
+RUN jupyter labextension install --no-build teradata_database* && \
+    jupyter labextension install --no-build teradata_resultset* && \
+    jupyter labextension install --no-build teradata_sqlhighlighter* && \
+    jupyter labextension install --no-build teradata_connection_manager* && \
+    jupyter labextension install --no-build teradata_preferences* && \
+    jupyter labextension install --no-build jupyterlab-plotly && \
```

Comment:
I would leave out the plotly extensions.  We include those in our docker but users don't need to include them.

Reply:
removed plotly.